docker/webapp.Dockerfile

```diff
@@ -1,9 +1,47 @@
+ARG APP_IMAGE=ocelotsocialnetwork/webapp
+ARG APP_IMAGE_TAG_BASE=latest-base
+ARG APP_IMAGE_TAG_CODE=latest-code
+ARG APP_IMAGE_BASE=${APP_IMAGE}:${APP_IMAGE_TAG_BASE}
+ARG APP_IMAGE_CODE=${APP_IMAGE}:${APP_IMAGE_TAG_CODE}
+
 ##################################################################################
-# BRANDED ########################################################################
+# CODE (branded) #################################################################
 ##################################################################################
-ARG APP_IMAGE=ocelotsocialnetwork/webapp:latest
-FROM $APP_IMAGE as branded
+FROM $APP_IMAGE_CODE as code
 
 # copy public constants into the Docker image to brand it
 COPY branding/static/ static/
 COPY branding/constants/ constants/
+
+##################################################################################
+# BUILD ##########################################################################
+##################################################################################
+FROM code as build
+
+# yarn install
+RUN yarn install --production=false --frozen-lockfile --non-interactive
+# yarn build
+RUN yarn run build
+
+##################################################################################
+# BRANDED (Does contain only "binary"- and static-files to reduce image size) ####
+##################################################################################
+FROM $APP_IMAGE_BASE as branded
+
+# TODO - do all copying with one COPY command to have one layer
+# Copy "binary"-files from build image
+COPY --from=build ${DOCKER_WORKDIR}/.nuxt ./.nuxt
+COPY --from=build ${DOCKER_WORKDIR}/node_modules ./node_modules
```

Comment:
No

Reply:
Is not working see main repo issue https://github.com/Ocelot-Social-Community/Ocelot-Social/issues/4471 .